### DISCUSSION TITLE
Publicando pacotes com dependências privadas

### DISCUSSION THREAD

**Discussion by luizgoebel** [balta-io/forum#243] [Tira Dúvidas] [answered]
fiz inumeras tentativas e não consigo achar onde estou errando
![image](https://user-images.githubusercontent.com/72111314/215282714-92b65778-8f86-4d83-b0ec-f9fd7e44055a.png)
![image](https://user-images.githubusercontent.com/72111314/215282729-2c097ecc-c8ed-4478-9871-65d441f397c3.png)
![image](https://user-images.githubusercontent.com/72111314/215282734-f0f76e05-d45b-4ded-98e3-8e2ca790f8ca.png)
![image](https://user-images.githubusercontent.com/72111314/215282754-f562d2de-c046-475f-b438-5deb554ad220.png)
![image](https://user-images.githubusercontent.com/72111314/215282779-173d22cb-9e15-4bb2-b802-9effaf5b0bc2.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @luizgoebel, como vai?

O erro diz que não conseguiu baixar o `MyCustomPrivatePackage`, mais especificamente que não conseguiu se autenticar no seu GitHub Package privado com o Token que você forneceu.

Você criou um Person Access Token e usou ele como o `TOKEN_GITHUB` (Cuidado para não expor este token).
👉 https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token

Você tem acesso para baixar o pacote do nuget (Acredito que sim, mas é bom revisar):
👉 https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry
